src/app/model-actions/model-actions.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="container">
+  <h2 *ngIf="edit"> Edit Model {{name}}</h2>
+  <h2 *ngIf="!edit"> Create a new model </h2>
+  <form [formGroup]="modelForm">
+    <div class="form-row">
+      <div class="form-group">
+        <label class="form-label"> Model Name: </label>
+        <input [ngClass]="{'is-invalid': formVariables.modelName.errors && formVariables.modelName.touched}" type="text" class="form-control"
```

Comment:
angular has ng-valid and ng-invalid classes already set, you don't need to create them yourself